.github/workflows/conda.yml

```diff
@@ -70,11 +73,7 @@ jobs:
 
       - name: Check installed toolkits
         run: |
-          # Checkout the state of the repo as of the last release (including RCs)
           export LATEST_TAG=$(git ls-remote --tags https://github.com/openforcefield/openff-toolkit.git | cut -f2 | grep -E "([0-9]+)\.([0-9]+)\.([0-9]+)$" | sort --version-sort | tail -1 | sed 's/refs\/tags\///')
-          git fetch --tags
-          git checkout tags/$LATEST_TAG
-          git log -1 | cat
```

Comment:
(blocking) This line seems essential to ensuring that the last release is being tested against its own tests, rather than the tests currently on `main`. Could you clarify?

Reply:
As above